### PR TITLE
add cn city_suffix field

### PR DESCRIPTION
**Pull request by CY-1**
add cn city_suffix field and test

### CODE REVIEW

File: src/main/resources/zh-CN.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ zh-CN:
   faker:
     address:
       city: [海门, 鄂尔多斯, 招远, 舟山, 齐齐哈尔, 盐城, 赤峰, 青岛, 乳山, 金昌, 泉州, 莱西, 日照, 胶南, 南通, 拉萨, 云浮, 梅州, 文登, 上海, 攀枝花, 威海, 承德, 厦门, 汕尾, 潮州, 丹东, 太仓, 曲靖, 烟台, 福州, 瓦房店, 即墨, 抚顺, 玉溪, 张家口, 阳泉, 莱州, 湖州, 汕头, 昆山, 宁波, 湛江, 揭阳, 荣成, 连云港, 葫芦岛, 常熟, 东莞, 河源, 淮安, 泰州, 南宁, 营口, 惠州, 江阴, 蓬莱, 韶关, 嘉峪关, 广州, 延安, 太原, 清远, 中山, 昆明, 寿光, 盘锦, 长治, 深圳, 珠海, 宿迁, 咸阳, 铜川, 平度, 佛山, 海口, 江门, 章丘, 肇庆, 大连, 临汾, 吴江, 石嘴山, 沈阳, 苏州, 茂名, 嘉兴, 长春, 胶州, 银川, 张家港, 三门峡, 锦州, 南昌, 柳州, 三亚, 自贡, 吉林, 阳江, 泸州, 西宁, 宜宾, 呼和浩特, 成都, 大同, 镇江, 桂林, 张家界, 宜兴, 北海, 西安, 金坛, 东营, 牡丹江, 遵义, 绍兴, 扬州, 常州, 潍坊, 重庆, 台州, 南京, 滨州, 贵阳, 无锡, 本溪, 克拉玛依, 渭南, 马鞍山, 宝鸡, 焦作, 句容, 北京, 徐州, 衡水, 包头, 绵阳, 乌鲁木齐, 枣庄, 杭州, 淄博, 鞍山, 溧阳, 库尔勒, 安阳, 开封, 济南, 德阳, 温州, 九江, 邯郸, 临安, 兰州, 沧州, 临沂, 南充, 天津, 富阳, 泰安, 诸暨, 郑州, 哈尔滨, 聊城, 芜湖, 唐山, 平顶山, 邢台, 德州, 济宁, 荆州, 宜昌, 义乌, 丽水, 洛阳, 秦皇岛, 株洲, 石家庄, 莱芜, 常德, 保定, 湘潭, 金华, 岳阳, 长沙, 衢州, 廊坊, 菏泽, 合肥, 武汉, 大庆]
+      city_suffix: [河北省,山西省,辽宁省,吉林省,黑龙江省,江苏省,浙江省,安徽省,福建省,江西省,山东省,河南省,湖北省,湖南省,广东省,海南省,四川省,贵州省,云南省,陕西省,甘肃省,青海省,台湾省,内蒙古自治区,广西壮族自治区,西藏自治区,宁夏回族自治区,新疆维吾尔自治区]
```
Comment (kingthorin) [inline src/main/resources/zh-CN.yml:6]: Staying consistent with entries above and below please use comma space.

File: src/test/java/net/datafaker/providers/base/AddressTest.java [hunk 2]
```diff
@@ -273,4 +273,11 @@ void dutchAddress() {
     void australiaAddress() {
         assertThat(AU_FAKER.address().fullAddress()).matches("(Unit|[0-9]).+, [A-Z].+, [A-Z]{2,3} [0-9]{4}");
     }
+
+    @RepeatedTest(100)
+    void cnAddressTest() {
```
Comment (kingthorin) [inline src/test/java/net/datafaker/providers/base/AddressTest.java:278]: This isn't testing address, it's testing suffix

**Comment by kingthorin** [comment]
Please remove the unrelated commit.

File: src/test/java/net/datafaker/providers/base/AddressTest.java (hunk 2 above)
Comment (CY-1) [reply src/test/java/net/datafaker/providers/base/AddressTest.java]: ok ，I deleted this code and remove the unrelated commit.。

**Comment by bodiam** [comment]
@CY-1 thanks for your contribution. Would it be possible to add a test which demonstrates this change?

**Comment by bodiam** [comment]
Thanks! This looks very reasonable to me! I'll merge it!

**Comment by bodiam** [comment]
It's available now for use in the snapshot version (2.0.2-SNAPSHOT) if you want to try it out.

**Comment by CY-1** [comment]
> It's available now for use in the snapshot version (2.0.2-SNAPSHOT) if you want to try it out.

thank you

**Pull request merged**